spec/features/thesis_spec.rb

```diff
@@ -19,10 +19,6 @@
       within 'dd.blacklight-restrictions_note_display' do
         find_link 'Mudd Manuscript Library'
       end
-      find('span', text: 'Mudd Manuscript Library')
```

Comment:
These assertions were failing (there are now multiple spans that contain the library name), and were not very meaningful, so I removed them.